components/clientComponents/forms/Form/Form.tsx

```diff
@@ -58,6 +64,11 @@ const InnerForm: React.FC<InnerFormProps> = (props) => {
   const groupsHeadingRef = useRef<HTMLHeadingElement>(null);
   const { getFormDelayWithGroups, getFormDelayWithoutGroups } = useFormDelay();
 
+  // @TODO add back the feature flag check when going to staging
```

Comment:
Remember to put the feature flag check back  before merging to staging. This is done because the feature flag won't exist until it's merged to staging.